var/spack/repos/builtin/packages/casacore/package.py

```diff
@@ -109,7 +115,16 @@ def cmake_args(self):
         else:
             args.extend(["-DBUILD_PYTHON=YES", "-DBUILD_PYTHON3=NO"])
 
-        args.append("-DBUILD_TESTING=OFF")
+        if spec.satisfies("@3.6.0:"):
+            args.append("-DUSE_PCH=ON")
```

Comment:
Consider using the `define` method for CMakePackages.
```suggestion
            args.append(self.define("USE_PCH", True))
```

Reply:
done